docs/source/ops/index.rst

```diff
@@ -0,0 +1,86 @@
+
+Please refer to the :doc:`protobuf documentation of the API </app-dev/grpc/proto-docs>` for details about the ``prune`` operation itself and the behavior of other DAML Ledger API endpoints when pruning is being or has been performed.
+
+Other limitations
+-----------------
+
+- If the number of divulged contracts continually grows, the storage used by the participant node may still keep growing, even with regular pruning.
+- Pruning may be rejected even if the node is running correctly (for example, to preserve non-repudiation properties); in this case, the application might not be able to archive contracts containing PII or pruning of these contracts may not be possible; thus, actually deleting this PII may also be technically unfeasible.
+- Pruning may leave parties, packages, and configuration data on the participant node, even if they are no longer needed for transaction processing, and even if they contain PII [3]_.
+- Pruning does not move pruned information to cold storage but simply deletes pruned data; for this reason, it is advisable to back up the Participant Index DB before invoking pruning. See the next sub-section for more Participant Index DB-related advice before and after invoking `prune`.
+- Pruning is not selective but rather effectively truncates the ledger, removing the transactions and archived contracts at and before the pruning offset.
+
+.. [3] This might be improved in future versions.
+
+Pruning and the DAML Participant Index DB
+-----------------------------------------
+
+Activities to be carried out *before* invoking a pruning operation include:
+
+- Backing up the Participant Index DB, as pruning will not move information to cold storage but rather it will delete archived contracts and transactions earlier than or at the pruning offset.
+
+Activities to be carried out *after* invoking a pruning operation include:
+
+- On a PostreSQL Index DB, a manual `VACUUM FULL` command is needed for the DB to give back disk space to the OS.
+
+Determining a suitable pruning offset
+-------------------------------------
+
+The :ref:`Active Contract Service <active-contract-service>` and the :ref:`Transaction Service <transaction-service>` provide offset information for transactions and for Active Contracts snapshots respectively: such offset can be used unchanged with `prune` calls.
+
+Scheduled jobs, applications and/or operator tools can be built on top of the DAML Ledger API to implement pruning automatically, for example at regular intervals, or on-demand, for example according to a user-initiated process.
+
+For instance, pruning at regular intervals could be performed by a cron job that:
+
+1. If a pruning interval has been saved to a well-known location:
+   a. Backs up the DAML Participant Index DB.
+   b. Performs pruning.
+   c. (If using PostgreSQL) Performs a `VACUUM FULL` command on the DAML Participant Index DB.
+
+2. Acquires a fresh Active Contract Set and saves the offset.
```

Comment:
This is for the purpose of knowing the offset to use on one of the subsequent pruning calls, right? I would highlight / track somewhere the use-case for helping the operator determine a suitable offset. E.g. in canton we currently use a [`locate_offset` helper](https://www.canton.io/docs/stable/user-manual/usermanual/console.html#ledger-pruning) which may or may not serve as inspiration for at least an "alpha/preview" level piece of api once we feel sufficiently sure about committing even to a slightly-more-deprecatable piece of admin api perhaps on the `ParticipantPruningService`.

Reply:
> This is for the purpose of knowing the offset to use on one of the subsequent pruning calls, right?

Exactly right.

> I would highlight / track somewhere the use-case for helping the operator determine a suitable offset.

@gerolf-da I have sent some notes to help us track this, let me know if I can help further.

Reply:
Typically the offset from the ActiveContractsService is fairly recent. Pruning right away would be too aggressive. I think it's worth pointing this out.

Reply:
Instead of using the ActiveContractsService to retrieve the offset, it's more efficient to query the ledger end (`TransactionService#GetLedgerEnd`) in regular intervals (e.g. daily at midnight) and store the corresponding offset.

Reply:
Yep, changed.